frontend/packages/client/src/components/Community/StrategyEditorModal/StrategySelector.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { kebabToString } from 'utils';
+
+export default function StrategySelector({
+  strategies = [],
+  onSelectStrategy,
+} = {}) {
+  const selectStrategy = (stratgy) => () => onSelectStrategy(stratgy);
+  return (
+    <div
+      className="is-flex is-flex-direction-column flex-1"
+      style={{ minHeight: '280px' }}
+    >
+      {strategies.map((st, index) => {
+        return (
+          <div
+            key={`strategy-${index}`}
+            className="border-light rounded-sm is-flex is-flex-direction-column is-justify-content-center mb-4 py-4 px-3 cursor-pointer strategy-selector transition-all"
+            style={{ minHeight: '99px' }}
+            onClick={selectStrategy(st.key)}
+          >
+            <div className="columns is-multiline">
+              <div className="column is-12 pb-2">
+                <p>{kebabToString(st.key)}</p>
```

Comment:
lets apply `text-transform: capitalize;` to the title of the strategies

Reply:
Can you use `capitalize` instead of uppercase? The difference is uppercase transforms all characters and capitalize will only transform the first letter of each word.

Reply:
<img width="470" alt="Screen Shot 2022-06-29 at 18 48 34" src="https://user-images.githubusercontent.com/7526740/176551313-6d2d25d2-4561-4e8a-a77b-33ea571fcda4.png">